### PR TITLE
Hold screen tests

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Add tests for Hold screen in Calling and CallWithChatComposites
# Why

Adds tests for new hold screen in composites

https://skype.visualstudio.com/SPOOL/_workitems/edit/2945286
# How Tested

Ran tests locally

### CODE REVIEW

File: packages/react-composites/src/composites/CallComposite/components/CallControls.tsx [hunk 1]
```diff
@@ -97,7 +99,7 @@ export const CallControls = (props: CallControlsProps & ContainerRectProps): JSX
   /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
   const holdButtonProps = usePropsFor(HoldButton);
 
-  /* @conditional-compile-remove(one-to-n-calling) */ /* @conditional-compile-remove(PSTN-calls) */
+  /* @conditional-compile-remove(one-to-n-calling) */ /* @conditional-compile-remove(one-to-n-calling) */
```
Comment (carocao-msft) [inline packages/react-composites/src/composites/CallComposite/components/CallControls.tsx:102]: Why are we adding two (one to n calling) here?

**Review by carocao-msft — COMMENTED** [review]
Looks good for now, waiting for snapshots

File: packages/react-composites/src/composites/CallComposite/components/CallControls.tsx (hunk 1 above)
Comment (dmceachernmsft) [reply packages/react-composites/src/composites/CallComposite/components/CallControls.tsx:102]: good question, must be a result of my merging conflicts, will fix

File: packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx [hunk 2]
```diff
@@ -82,6 +82,9 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
   /* @conditional-compile-remove(one-to-n-calling) */
   const { callStatus } = useSelector(callStatusSelector);
 
+  /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
+  const disableControlsForHoldScreen = adapter.getState().page === 'hold';
```
Comment (anjulgarg) [inline packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx:86]: this shouldn't be here anymore

File: packages/react-composites/src/composites/CallComposite/components/CallControls.tsx [hunk 3]
```diff
@@ -51,6 +51,8 @@ export type CallControlsProps = {
    */
   increaseFlyoutItemSize?: boolean;
   isMobile?: boolean;
+  /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
```
Comment (anjulgarg) [inline packages/react-composites/src/composites/CallComposite/components/CallControls.tsx:54]: this shouldn't be here anymore

File: packages/react-composites/src/composites/CallComposite/components/CallControls.tsx [hunk 4]
```diff
@@ -114,7 +116,7 @@ export const CallControls = (props: CallControlsProps & ContainerRectProps): JSX
         itemProps: {
           styles: buttonFlyoutIncreasedSizeStyles
         },
-        ['data-ui-id']: 'call-composite-more-menu-people-button'
+        ['data-ui-id']: 'people-button-call-composite-mobile'
```
Comment (anjulgarg) [inline packages/react-composites/src/composites/CallComposite/components/CallControls.tsx:119]: Let's use the existing name which is consistent with other names

**Review by anjulgarg — APPROVED** [review]

**Review by carocao-msft — APPROVED** [review]

**Pull request merged**